### ISSUE TITLE
关于DBNet蒸馏CML方法的一些疑问

### ISSUE THREAD

**Issue by DYJNG** [PaddlePaddle/PaddleOCR#6375]
您好，看到CML方法中教师模型的监督是用教师模型输出的shrink_map做二值化+膨胀得到。我有以下两个疑问：
1. 为什么要做膨胀操作，这么做的原理是什么呢？
2. 为什么不直接用教师模型输出的shrink_map作为软标签监督学生模型的shrink_map，二值化后shrink_map监督binary_map呢？
麻烦大佬得空帮忙解惑，谢谢！

**Comment by LDOUBLEV** [comment]
1. 膨胀操作是为了提升端对端精度，可视化二值图计算得到的框发现有些框虽然比较契合文字位置，但是膨胀后能更准确的识别。
2. 作为监督的teacher shrink_map其实是二值化后的：https://github.com/PaddlePaddle/PaddleOCR/blob/453ae6bbb3e15c141eaeb418a00fc5a21601555b/ppocr/losses/distillation_loss.py#L268  至于不用软标签的原因是，我发现shrink_map中的像素小于threshold的值会出现一些"噪点"，所以强制把< threshold的部分设置为0防止影响训练，而大于threshold的像素90%都是>0.8的；所以直接用二值化了；

**Comment by LDOUBLEV** [comment]
感兴趣你也可以在你的数据上做一些消融试验，有问题一起讨论

**Comment by DYJNG** [comment]
> 1. 膨胀操作是为了提升端对端精度，可视化二值图计算得到的框发现有些框虽然比较契合文字位置，但是膨胀后能更准确的识别。
> 2. 作为监督的teacher shrink_map其实是二值化后的：https://github.com/PaddlePaddle/PaddleOCR/blob/453ae6bbb3e15c141eaeb418a00fc5a21601555b/ppocr/losses/distillation_loss.py#L268
>      至于不用软标签的原因是，我发现shrink_map中的像素小于threshold的值会出现一些"噪点"，所以强制把< threshold的部分设置为0防止影响训练，而大于threshold的像素90%都是>0.8的；所以直接用二值化了；

感谢大佬解答！ 
因为我在自己的数据集上实验，加上膨胀蒸馏对于检测指标几乎没有提升，而去掉膨胀蒸馏，检测指标反而会有一定的提升，所以才有这个疑惑。原来是对识别有利，明白了~

**Comment by OneYearIsEnough** [comment]
> 1. 膨胀操作是为了提升端对端精度，可视化二值图计算得到的框发现有些框虽然比较契合文字位置，但是膨胀后能更准确的识别。
> 2. 作为监督的teacher shrink_map其实是二值化后的：https://github.com/PaddlePaddle/PaddleOCR/blob/453ae6bbb3e15c141eaeb418a00fc5a21601555b/ppocr/losses/distillation_loss.py#L268
>      至于不用软标签的原因是，我发现shrink_map中的像素小于threshold的值会出现一些"噪点"，所以强制把< threshold的部分设置为0防止影响训练，而大于threshold的像素90%都是>0.8的；所以直接用二值化了；

把teacher输出二值化后，如果二值化后的效果比较理想，相当于又和gt做了一遍db loss，CML就退化成了DML……为什么不是student和teacher间的logits做KL Loss呀。。
